addons/website_sale_product_configurator/views/website_sale_product_configurator_templates.xml

```diff
@@ -4,6 +4,14 @@
         <xpath expr="//th[hasclass('td-qty')]/span[hasclass('label')]" position="attributes">
             <attribute name='t-if'>not request.is_frontend or (request.is_frontend and is_view_active('website_sale.product_quantity'))</attribute>
         </xpath>
+        <xpath expr="//td[hasclass('td-price')]" position="inside">
+            <div t-if="combination_info.get('compare_list_price')" t-attf-class="text-danger oe_striked_price {{'' if combination_info['compare_list_price'] else 'd-none'}}"
+                t-out="combination_info['compare_list_price']"
+                t-options='{
+                    "widget": "monetary",
+                    "display_currency": (pricelist or product).currency_id
+                }'/>
+        </xpath>
```

Comment:
There's already a striked price in the overridden template, so we should probably hide that one if we show this one. I think `compare_list_price` takes precedence but you may want to double-check what we do in the product page.

Also, I think we should check whether `combination_info['compare_list_price'] > combination_info['price']`, otherwise we probably shouldn't show it.

Lastly, could you also apply these changes to the `optional_product_items_website` template below?

Reply:
Yes, there is already a strike price in the overridden template. I took a look at how the pricing is implemented on the product page and I'm going to use a similar logic here to align it with that. This should address all the points you mentioned.

Regarding the `optional_product_items_website` , I am currently on it.